### PR TITLE
feat: make a position page and a pool page

### PR DESCRIPTION
**Pull request by reallybeard**
## Description

- Splits the pool component into a position and pool view
## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes #

### Operations

No testing required, just UI.

## Screenshots (if applicable)
Screenshot 2024-01-17 at 3 13 48 PM

### CODE REVIEW

File: src/pages/ThorChainLP/Pool/components/PairRates.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  const asset1MarketData = useAppSelector(state => selectMarketDataById(state, assetIds[1]))
+
+  const asset0PricePerAsset1 = bn(asset0MarketData.price)
+    .div(asset1MarketData.price)
+    .div(1)
+    .toString()
+  const asset1PricePerAsset0 = bn(asset1MarketData.price)
+    .div(asset0MarketData.price)
+    .div(1)
+    .toString()
```
Comment (woodenfurniture) [inline src/pages/ThorChainLP/Pool/components/PairRates.tsx:60]:
```suggestion
  const asset0PricePerAsset1 = bn(asset0MarketData.price)
    .div(asset1MarketData.price)
    .toString()
  const asset1PricePerAsset0 = bn(asset1MarketData.price)
    .div(asset0MarketData.price)
    .toString()
```

Comment (woodenfurniture) [reply src/pages/ThorChainLP/Pool/components/PairRates.tsx:60]: x / 1 = x

**Review by woodenfurniture — APPROVED** [review]
1 non-blocking comment, but lgtm

File: src/pages/ThorChainLP/Pool/components/PairRates.tsx (hunk 1 above)
Comment (0xApotheosis) [reply src/pages/ThorChainLP/Pool/components/PairRates.tsx]: 🧠

**Pull request merged**